app/helpers/maintain.py

```diff
@@ -0,0 +1,92 @@
+import configparser
+from pathlib import Path
+from app.config import (
+    DATA_DIR,
+    CLI_FETCH_CLEAN_ENTRY_NAMES,
+)
+from datetime import date, datetime, timedelta
+from dateutil.relativedelta import relativedelta
+from importlib import import_module
+
+HISTORY_CONFIG_FILENAME = "maintenance_history.cfg"
+DATA_CONFIG_FILENAME = "maintenance.cfg"
```

Comment:
What's the reasoning for separating these out into two files? Could we just store them in a single `maintenance.cfg` file?

Reply:
Ah good call, that makes sense! I think I'm hoping to avoid clutter on this top level of the directory. What if we went with the following system:
1. There's a `maintenance.cfg` file at the top level of the directory, which is tracked by git and stores data on the required refresh interval for the pipeline.
2. Whenever that pipeline's `fetch` or `clean` function is run, a little `timestamp.cfg` or `history.cfg` (or whatever name you think is better) file is generated and dropped into the `data` or `raw-data` folder. For example, if we run the `clean` function, then the file at `data/timestamp.cfg` is clobbered with a new file containing the current date and time. These files will not be tracked by git.

Does that seem fair? I think I like that this puts the relevant metadata closer to the data it actually describes, it avoids clutter at the top-level, and it still allows us to achieve all the git tracking & non-tracking we desire. I guess one drawback is that anyone iterating through those directories will have to know they're there, but we should already be writing any iterative functions like that anyways.